### PR TITLE
Opt out of article count when the user removes consent

### PR DESCRIPTION
**Pull request by tompretty**
## What does this change?

Opt users out of article count epics/banners if we don't have TCFv2 consent, as per [this card](https://trello.com/c/5QeO8WKw/2313-do-not-show-article-count-to-users-who-have-not-consented-to-the-cmp). This is basically a copy of [this pr](https://github.com/guardian/dotcom-rendering/pull/1975) in dcr.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] Yes 

## Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by mxdvl — CHANGES_REQUESTED** [review]
Looks good!

Caught a few typos.

File: static/src/javascripts/projects/common/modules/commercial/contributions-service.js [hunk 1]
```diff
@@ -81,7 +82,44 @@ const epicEl = () => {
     return container;
 }
 
-const buildEpicPayload = () => {
+const hasOptedOutOfArticleCount = () =>
+    !!getCookie(ARTICLES_VIEWED_OPT_OUT_COOKIE.name)
+
+const DAILY_ARTILCE_COUNT_KEY = 'gu.history.dailyArticleCount';
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/commercial/contributions-service.js:88]: Change ARTILCE &rarr; ARTICLE

```suggestion
const DAILY_ARTICLE_COUNT_KEY = 'gu.history.dailyArticleCount';
```

Comment (TomPretty) [reply static/src/javascripts/projects/common/modules/commercial/contributions-service.js]: good spot! 👍

**Review by mxdvl — APPROVED** [review]

**Review by tomrf1 — APPROVED** [review]
💯

**Pull request merged**